mkdocs.yml

```diff
@@ -110,8 +111,15 @@ extra:
       link: https://dietpi.com/
     - icon: fontawesome/brands/twitter
       link: https://twitter.com/DietPi_
+  item076:
```

Comment:
Maybe `software076` so that we can use `hardware076` for a certain hardware/SBC model in the future?

Reply:
it's a great suggestion - I have created a software file